src/extension/atlas/atlas_unwrap.cpp

```diff
@@ -0,0 +1,202 @@
+/**************************************************************************/
```

Comment:
Is this actually code copy-pasted from the engine? Is there no way to call this method through the GDExtension interface?

Reply:
It is copy-pasted from the engine; it isn't possible to call the original method, because it's heavily tied to lightmaps. Additionally, we might want to make some changes to its functionality in future, so I think that it makes to keep it as a separate method

Reply:
Alright, yep it looks like it is not exposed from the engine (looking in the godot-cpp project). So I guess this needs to be done like this.